script.skin.helper.service/addon.xml

```diff
@@ -10,22 +10,34 @@
 	</requires>
 	
     <extension point="xbmc.python.library" library="default.py" />
-	<extension point="xbmc.python.pluginsource" library="plugin.py">
-		<provides>video audio</provides>
-	</extension>
+	<extension point="xbmc.python.pluginsource" library="plugin.py" />
 	<extension library="service.py" point="xbmc.service" start="login" />
-	<extension point="kodi.context.item" library="contextmenu_serieslevel.py">
+	<extension point="kodi.context.item" library="resources\lib\ContextMenu_serieslevel.py">
 		<item>
 			<label>32072</label>
             <description>Display Series level button in next/recent/in progress episodes listings</description>
-			<visible>Container.Content(episodes) + !StringCompare(Container.ListItem(0).TvShowTitle,Container.ListItem(1).TvShowTitle)</visible>
+			<visible>!IsEmpty(Window(Home).Property(SkinHelper.enablecontextmenu_series)) + Container.Content(episodes) + !StringCompare(Container.ListItem(0).TvShowTitle,Container.ListItem(1).TvShowTitle)</visible>
+		</item>
+	</extension>
+    <extension point="kodi.context.item" library="resources\lib\ContextMenu_pvr.py">
+		<item>
+			<label>32120</label>
+            <description>PVR artwork Configuration</description>
+            <visible>!IsEmpty(Window(Home).Property(SkinHelper.enablecontextmenu_pvr)) + [Window.IsActive(MyPVRChannels.xml) | Window.IsActive(MyPVRGuide.xml) | Window.IsActive(MyPVRRecordings.xml) | Window.IsActive(MyPVRTimers.xml) | Window.IsActive(MyPVRSearch.xml)] + Skin.HasSetting(SkinHelper.EnablePVRThumbs)</visible>
```

Comment:
The contextmenu pops up a dialog with the specific option for that feature.
Each feature is enabled by the skinner by setting a skin bool. Also, the contextmenu item can be disabled by the user in the addon settings.

For example, the PVR artwork. If enabled by the skinner, the user is able to do some overrides if the auto detection was off or just set the artwork manually.

As we discussed on Github last week I will most likely migrate all code into seperate (module) addons in the near future.